ValidateCommitMessage.yml

```
@@ -17,25 +17,85 @@ jobs:
-      source venv/bin/activate
-      pip install --upgrade pip setuptools wheel
-      python setup.py sdist
-    displayName: 'Validate pip package build'
\ No newline at end of file
+  - bash: |
+      echo "Installing PowerShell Core..."
+      wget -q https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb
+      sudo dpkg -i packages-microsoft-prod.deb
+      sudo apt-get update
+      sudo apt-get install -y powershell
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)